.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Setup Node
+      uses: actions/setup-node@v3.1.1
+      with:
+        node-version: 16.13
+        cache: npm
+        
+    - name: Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
```

Comment:
@linguists rust toolchain version is specified here, latest stable version by default.

Reply:
oh, cool. I think it's OK.